src/pages/Productdetail/ImageSlider.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './ImageSlider.scss';
+
+class ImageSlider extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sliderIndex: 0,
+      productSlideImg: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/productDetail.json', {})
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productSlideImg: data,
+        });
+      });
+  }
+
+  onChangeDetailImg = index => {
+    if (this.state.productSlideImg.length <= index) index = 0;
+    if (index < 0) index = this.state.productSlideImg.length - 1;
```

Comment:
매개변수로 받은 값을 다시 재할당하는 것보다는 새로운 변수를 하나 만들어서 활용하는게 좀 더 가독성측면에서 좋아보입니다.
```suggestion
  onChangeDetailImg = Index => {
     let nextSlideIndex = index;
     if (this.state.productSlideImg.length <= index) nextSlideIndex = 0;
     if (index < 0) nextSlideIndex = this.state.productSlideImg.length - 1;
```

Reply:
네 수정했습니다! 감사합니다!